Scripts/MNISTdataset.swift

```diff
@@ -0,0 +1,97 @@
+/*
+Usage:
+
+downloadAndExtract()
+let (trainX, trainY, testX, testY) = readMNIST(trainImages: constants["trainImages"]!, 
+                                                   trainLabels: constants["trainLabels"]!, 
+                                                   testImages: constants["testImages"]!, 
+                                                   testLabels: constants["testLabels"]!)
+*/
+
+import Python
+import TensorFlow
+
+let np = Python.import("numpy")
+let path = Python.import("os.path")
+let urllib = Python.import("urllib.request")
+let gzip = Python.import("gzip")
+let sh = Python.import("shutil")
+
+// MNIST constants
+let constants: [String: String] = [
```

Comment:
> Declaring an enum without cases is the canonical way to define a “namespace” to group a set of related declarations, such as constants or helper functions.

(swift style guide)

An advantage over this `[String: String]` approach is that the compiler will catch references to missing constants at compile time!

The enum should be `fileprivate`.